cola/widgets/selectcommits.py

```diff
@@ -135,6 +143,14 @@ def commit_oid_double_clicked(self, _item):
         if oid:
             self.accept()
 
+    def search_list(self, text):
+        if text:
+            for i in range(self.commits.count()):
+                self.commits.item(i).setHidden(True) 
```

Comment:
pylint says:
```
cola/widgets/selectcommits.py:149:52: Trailing whitespace (trailing-whitespace)
Makefile:292: recipe for target 'pylint' failed
```

Maybe instruct your editor to remove trailing spaces on save?

Reply:
Yeah will have to do that :D